apps/bbsync/srtnotes.py

```diff
@@ -176,12 +176,17 @@ def generate_jira_trackers(self):
     def generate_references(self):
         """
         generate array of references to SRT notes
+
+        OSIDB uses "ARTICLE" and "EXTERNAL",
+        but Bugzilla uses "vuln_response" and "external".
         """
         self.add_conditionally(
             "references",
             [
                 {
-                    "type": reference.type,
+                    "type": "vuln_response"
+                    if reference.type == "ARTICLE"
+                    else "external",
```

Comment:
nitpick: I would rather see there a map to make it more future proof but so far with the two values it is not really necessary. You can ignore this comment.